src/test/TxTests.cpp

```diff
@@ -20,7 +20,7 @@
 #include "transactions/CreateAccountOpFrame.h"
 #include "transactions/InflationOpFrame.h"
 #include "transactions/ManageDataOpFrame.h"
-#include "transactions/ManageOfferOpFrame.h"
```

Comment:
should also apply adds/renames to the Visual Studio files

Reply:
Fixed. Also added ManageBuyOfferOpFrame and ManageOfferOpFrameBase.